cast/src/lib.rs

```diff
@@ -1050,6 +1054,21 @@ impl SimpleCast {
         Ok(format!("0x{}{}", "0".repeat(64 - num_hex.len()), num_hex))
     }
 
+    pub fn left_shift(value: &str, bits: &str, base_in: u32) -> Result<U256> {
+        dbg!(&base_in);
+        let value = U256::from_str_radix(value, base_in).unwrap();
+        let bits = U256::from_str_radix(bits, base_in).unwrap();
```

Comment:
Use the results instead. You can provide better context to the user with `wrap_err`: https://docs.rs/eyre/latest/eyre/trait.WrapErr.html#tymethod.wrap_err